app/main/index.js

```diff
@@ -170,11 +174,12 @@ function createMainWindow() {
 		icon: iconPath(),
 		minWidth: 600,
 		minHeight: 400,
+		titleBarStyle: 'hidden-inset',
 		webPreferences: {
 			preload: path.join(__dirname, '../renderer/js/preload.js'),
 			plugins: true,
 			allowDisplayingInsecureContent: true,
-			nodeIntegration: false
+			nodeIntegration: true
```

Comment:
@akashnimare I seem to recall that enabling `nodeIntegration` caused weird problems with the webapp's use of jQuery from node; do you remember the details?  Regardless, we should add a comment documenting why it's disabled if we need it to still be disabled, so folks know there's a reason.